pom.xml

```diff
@@ -91,6 +129,26 @@
                             <name>${dockerCafImagePrefix}opensuse-tomcat${dockerProjectVersion}</name>
                             <build>
                                 <dockerFileDir>.</dockerFileDir>
+                                 <assembly>
+                                    <basedir>/</basedir>
+                                    <inline>
+                                        <dependencySets>
+                                            <dependencySet>
+                                                <includes>
+                                                    <include>ch.qos.logback:logback-access</include>
+                                                    <include>ch.qos.logback:logback-core</include>
+                                                    <include>ch.qos.logback:logback-classic</include>
+                                                    <include>com.github.cafapi.logging:caf-logging-common</include>
+                                                    <include>com.github.cafapi.logging:caf-logging-logback-access</include>
+                                                    <include>com.github.tomcat-slf4j-logback:tomcat9-slf4j-logback</include>
+                                                    <include>com.github.cafapi.util:util-process-identifier:jar</include>
+                                                </includes>
```

Comment:
Why is this so complicated?  Isn't it just all dependencies?

Reply:
This still looks over-complicated.  Are the `includes` and `useTransitiveFiltering` elements really needed?